src/libraries/System.Private.CoreLib/src/System/IO/Directory.cs

```diff
@@ -52,6 +53,29 @@ public static DirectoryInfo CreateDirectory(string path)
         public static DirectoryInfo CreateDirectory(string path, UnixFileMode unixCreateMode)
             => CreateDirectoryCore(path, unixCreateMode);
 
+        /// <summary>
+        /// Creates a uniquely-named, empty directory in the current user's temporary directory.
+        /// </summary>
+        /// <param name="prefix">An optional string to add to the beginning of the subdirectory name.</param>
+        /// <returns>An object that represents the directory that was created.</returns>
+        /// <exception cref="ArgumentException"><paramref name="prefix" /> contains a directory separator.</exception>
+        /// <exception cref="IOException">A new directory cannot be created.</exception>
+        public static unsafe DirectoryInfo CreateTempSubdirectory(string? prefix = null)
```

Comment:
```suggestion
        public static unsafe DirectoryInfo CreateTempSubDirectory(string? prefix = null)
```

Is it `Subdirectory` or `SubDirectory`?

Reply:
`Subdirectory` - see https://docs.microsoft.com/en-us/dotnet/api/system.io.directoryinfo.createsubdirectory

Reply:
Ahh... Even that should've been `SubDirectory` but too late it seems.

Reply:
https://www.merriam-webster.com/dictionary/subdirectory

says it is one word.

Reply:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/4962580/183507074-66ead510-3524-468e-85fb-ddf4cf7bf41c.png">

It may be less common but in the programming world, especially in naming public APIs, the naming has to highlight individual words and acronyms. Of course, not every complex/compound word falls into this.

Reply:
I sen so not think there are many examples in core .NET API where we capitalize a letter in the middle of what is generally a single word. It would surprise me as a user.